src/pages/Learn/Lesson.jsx

```diff
@@ -0,0 +1,25 @@
+import styles from './Lesson.module.css';
+import PropTypes from 'prop-types';
+
+const Lesson = ({ lesson }) => {
+  const { title, materials = '' } = lesson;
+  // const { title, materials = '', videoUrl = [] } = lesson;
```

Comment:
are we comment it out because of the other pr?

Reply:
Lesson page is for other pr.